rapids-cmake/cpm/versions.json

```
@@ -66,20 +65,13 @@
     },
     "rmm": {
       "version": "${rapids-cmake-version}",
-      "git_url": "https://github.com/rapidsai/rmm.git",
-      "git_tag": "branch-${version}"
+      "git_url": "https://github.com/jameslamb/rmm.git",
+      "git_tag": "fmt-and-spdlog"
```

```suggestion
      "git_url": "https://github.com/rapidsai/rmm.git",
      "git_tag": "branch-${version}"
```

TODO: revert this back right before merging